testing/tests/ui/as-primitive-type.stderr

```diff
@@ -1,59 +1,65 @@
 error: `as` operator expects the name of a primitive type on its right-hand side
-       failed to parse template source at row 1, column 8 near:
-       "as 4567 }}"
- --> tests/ui/as-primitive-type.rs:3:10
+ --> <source attribute>:1:9
   |
-3 | #[derive(Template)]
-  |          ^^^^^^^^
+       1 | {{ 1234 as 4567 }}
```

Comment:
Btw, is it normal to have this super wide space starting here?

Reply:
It is aligned to start after "error: ", because it belongs to the rust source that is printed below. At first, I did not find out how to remove the indentation, now I kind like it, because it given you a visual "this happened because of that" hint.

Reply:
I have to admit I hate it. ^^'

I'd prefer if we kept coherency with rustc (with columns aligned to the one with the biggest line error number):

```
$ rustc foo.rs 
error[E0425]: cannot find value `Ba` in this scope
  --> foo.rs:10:13
   |
7  | pub struct Bar;
   | --------------- similarly named unit struct `Bar` defined here
...
10 |     let x = Ba;
   |             ^^ help: a unit struct with a similar name exists: `Bar`

error: aborting due to 1 previous error
```